test/e2e/e2e_test.go

```diff
@@ -66,41 +61,37 @@ func TestOperator(t *testing.T) {
 		return string(output), nil
 	}
 	// Deploying the controller-manager
-	_, err = kind("make", "deploy", "install", fmt.Sprintf("IMG=%s", img))
+	_, err = kind("skaffold", "run", "--wait-for-connection=true", "-p", "integration")
+	require.NoError(t, err)
+	// Installing the CRDs
+	_, err = kind("make", "install")
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		_, err = kind("make", "undeploy", "ignore-not-found=true")
+		require.NoError(t, err)
+	})
 	// Accept the EULA and set the PID
 	_, err = kind("kubectl", "set", "env",
 		"-n", nsController, controller,
 		"MSSQL_ACCEPT_EULA=Y", "MSSQL_PID=Developer")
 	require.NoError(t, err)
-	// Restarting the controller-manager, to ensure the latest image is used
-	_, err = kind("kubectl", "rollout", "restart",
-		"-n", nsController, controller)
-	require.NoError(t, err)
-	// Waiting for the controller-manager to be ready
-	_, err = kind("kubectl", "rollout", "status",
-		"-n", nsController, controller,
-		"--timeout", "2m")
-	require.NoError(t, err)
```

Comment:
Replace with `skaffold`, it handle docker build, install resources, and check status